### PR TITLE
🚀 Migrate APIs to v2 

### LINKED ISSUE #235: Design a Card Component for v2 APIs
📍 Design a Card Component for v2 APIs

### Todo 🚀

😿 - Not Needed in the FE.

- [x] "id"
- [x] ~"state_name"~ 😿
- [x] "district_name"
- [x] ~"data_source"~ 😿
- [x] ~"data_name"~ 😿
- [x] ~"external_id"~ 😿
- [x] ~"created_date"~ 😿
- [x] ~"modified_date"~ 😿
- [x] ~"deleted"~ 😿
- [x] ~"data_id"~ 😿
- [x] "title"
- [x] "category"
- [x] "resource_type"
- [x] "description"
- [x] "address"
- [x] "phone_1"
- [x] "phone_2"
- [x] "email"
- [x] "quantity_available"
- [x] "price"
- [x] "source_link"
- [x] "comment"
- [x] ~"created_on"~ 😿
- [x] "created_by"~ 😿
- [x] ~"verified_by"~ 😿
- [x] "last_verified_on"
- [x] "downvotes"
- [x] "upvotes"
- [x] "created_job"
- [x] "district"
- [x] ~"state"~ 😿

### LINKED ISSUE #237: 📍 Route to v2 APIs
https://life-api.coronasafe.network/#api-v2-csv

In link with #235 📌

### PR DESCRIPTION
**Pull request by shrihari689**
## Proposed Changes 😺

- Route all APIs to v2 🔁
- Changed Resource Card to adopt different resource types. 📌

### Hospital
image

### Oxygen
image

## Known Bugs / Pending from Backend 😿
- [ ] ~~Active Districts json has duplicate id and records which may result in Duplicate Suggestions in Search 🐱~~
- [ ] ~~All Helpline json Phone Numbers are invalid~~
- [ ] ~~Mostly all resources don't have `category` field though it not an issue, but it should be fixed 📍~~
- [ ] ~~Need API endpoint for Upvotes and Downvotes 😪~~
- [ ] ~~Need to confirm as there is data since Hospital Related fields are empty.~~

Until these becomes stable 👆, we might need to put this PR in draft 🌲

🎉 Do review and comment, will the push the updates 🤗

### CODE REVIEW

**Comment by karmakoder** [comment]
@shrihari689 
- Active Districts json has duplicate id and records which may result in Duplicate Suggestions in Search :cat: - Thats ok to launch.
- All Helpline json Phone Numbers are invalid - Can we not show invalid numbers?
- Mostly all resources don't have category field though it not an issue, it should be fixed :round_pushpin: - Thats ok to launch.
- Need API endpoint for Upvotes and Downvotes :sleepy: - Thats ok to launch.
Key is to light up this data in UI and keep iterating/improving.

**Comment by shrihari689** [comment]
Okiee 📌 Then for now ignoring the above cons and shipping the UI 🚢

Note:

📍 Helpline uses our v1 API because of invalid phone numbers in v2
📍 There are some duplications coming from `active_district_data_v2.json`
📍 Some resources have category as null 

Everything is working fine here 😺

File: lib/api.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
-import ambulancesObjects from '@data/ambulance.json';
+import ambulancesObjects from '@data/ambulance_v2.json';
 import stateObjects from '@data/states.json';
-import oxygenObjects from '@data/oxygen.json';
-import medicineObjects from '@data/medicine.json';
-import hospitalObjects from '@data/hospital_clinic_centre.json';
+import oxygenObjects from '@data/oxygen_v2.json';
+import medicineObjects from '@data/medicine_v2.json';
+import hospitalObjects from '@data/hospital_v2.json';
 import helplineObjects from '@data/helpline.json';
```
Comment (bodhish) [inline lib/api.js:6]: This will break.

Comment (shrihari689) [reply lib/api.js]: 📍 Hope this will be fixed 😁

**Pull request merged**